webapp/loggedin/surveyViews/labelsEditor/labelsEditor.scss

```diff
@@ -1,22 +1,20 @@
-@import "~@webapp/style/vars";
+@import '~@webapp/style/vars';
 
 .labels-editor {
-
   display: grid;
-  grid-template-columns: 25px 1fr;
```

Comment:
labels editor without labels is used only in Collect Import Report: only one column is needed in the grid template